components/channel_header/channel_header.js

```diff
@@ -506,20 +506,25 @@ export default class ChannelHeader extends React.PureComponent {
                     className='channel-header__top'
                 >
                     {toggleFavorite}
-                    <strong
-                        id='channelHeaderTitle'
-                        className='heading'
+                    <button
+                        className='channel-header__trigger style--none'
+                        aria-label={formatMessage({id: 'channel_header.menuAriaLabel', defaultMessage: 'Channel Menu'})}
```

Comment:
@asaadmahmood shouldn't you be using `toLowerCase()` here?

Reply:
Yup, good point.